osu.Game/osu.Game.csproj

```diff
@@ -29,13 +29,14 @@
     <PackageReference Include="Microsoft.EntityFrameworkCore.Sqlite" Version="5.0.14" />
     <PackageReference Include="Microsoft.EntityFrameworkCore.Sqlite.Core" Version="5.0.14" />
     <PackageReference Include="Microsoft.Extensions.Configuration.Abstractions" Version="6.0.0" />
+    <PackageReference Include="Microsoft.Toolkit.HighPerformance" Version="7.1.2" />
```

Comment:
What is this? I hope this isn't just for the `.AsStream()` method...?

Reply:
That’s correct. I’m open to alternatives but it’s a pretty lightweight extension library.